vm-setup/roles/v1aX_integration_test/templates/controlplane_ubuntu.yaml

```diff
@@ -156,7 +170,7 @@ spec:
       - curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
       - echo 'deb https://apt.kubernetes.io/ kubernetes-xenial main' > /etc/apt/sources.list.d/kubernetes.list
       - apt update -y
-      - apt install docker-ce docker-ce-cli containerd.io kubelet kubeadm kubectl -y
+      - apt install docker-ce docker-ce-cli containerd.io kubelet={{KUBERNETES_BINARIES_VERSION}} kubeadm={{KUBERNETES_BINARIES_VERSION}} kubectl={{KUBERNETES_BINARIES_VERSION}} -y
```

Comment:
this should be replaced too, no ?

Reply:
indeed. 
I have replaced all on my text editor. 
Thanks, will check again.